src/widget/widget.ts

```diff
@@ -286,16 +352,21 @@ function createOutput({ URL_BASE }: SettingConstants) {
         >
           <div class="${style.selector_dateAndTimeSelectors}">
             <div class="${style.selector_dateSelector}">
-              <label for="searchTimeSelector-date">Dato</label>
+              <label for="${`${prefix}-searchTimeSelector-date`}">Dato</label>
               <input
                 type="date"
-                id="searchTimeSelector-date"
-                value="2023-11-10"
+                name="dateinput"
```

Comment:
Would this name ever conflict with other names on the page it is used?

Reply:
names are scoped to forms